storage/host-flash/host-flash.c

```diff
@@ -15,155 +15,167 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <unistd.h>
+#include <string.h>
 
 #include "host-flash.h"
 
 static struct {
-    int filefd;
-    size_t flashsz;
-    size_t sectorsz;
+	int filefd;
+	size_t flashsz;
+	size_t sectorsz;
 } hostflash_common;
 
 ssize_t hostflash_read(unsigned int addr, void *buff, size_t bufflen)
 {
-    ssize_t stat;
-    int read = 0;
+	ssize_t stat;
+	int read = 0;
 
-    if (addr + bufflen > hostflash_common.flashsz)
-        return -EINVAL;
+	if (addr + bufflen > hostflash_common.flashsz)
+		return -EINVAL;
 
-    while ((stat = pread(hostflash_common.filefd, (void *)((char *)buff + read), bufflen - read, addr + read)) != 0) {
-        if (stat < 0) {
-            if (errno == -EINTR)
-                continue;
+	while ((stat = pread(hostflash_common.filefd, (void *)((char *)buff + read), bufflen - read, addr + read)) != 0) {
+		if (stat < 0) {
+			if (errno == EINTR)
+				continue;
 
-            return stat;
-        }
+			return stat;
+		}
 
-        read += stat;
-        if (read == bufflen)
-            break;
-    }
+		read += stat;
+		if (read == bufflen)
+			break;
+	}
 
-    return read;
+	return read;
 }
 
 
 ssize_t hostflash_write(unsigned int addr, void *buff, size_t bufflen)
 {
-    char currentByte = 0;
-    size_t wrote = 0;
-    ssize_t stat;
-
-    if (addr + bufflen > hostflash_common.flashsz)
-        return -EINVAL;
-
-    while (wrote != bufflen) {
-        stat = hostflash_read(addr + wrote, &currentByte, 1);
-        if (stat <= 0)
-            return stat;
-
-        currentByte = currentByte & *((char *)buff + wrote);
-        stat = pwrite(hostflash_common.filefd, &currentByte, 1, addr + wrote);
-        if (stat < 0) {
-            if (errno == -EINTR)
-                continue;
-
-            return stat;
-        }
-
-        wrote++;
-    }
-
-    return (ssize_t)wrote;
+	char tempTab[256];
+	size_t wrote = 0, i;
+	int len;
+	ssize_t stat, readed;
```

Comment:
`readed` is not a valid word in english :) The past tense of read is read

Reply:
english language is so hard sometimes :) Done